Services/Library/LibraryService.cs

```diff
@@ -1,8 +1,10 @@
 ﻿using Acuedify.Data;
 using Acuedify.Models;
 using Acuedify.Services.Library.Interfaces;
+using Acuedify.Services.Playing.Interfaces;
 using Microsoft.EntityFrameworkCore;
 using Microsoft.IdentityModel.Tokens;
+using System.Text.RegularExpressions;
```

Comment:
I assume these using statements are not needed as there are no other changes in file?

Reply:
no, forgot to remove when I decided to move the functions needed elsewhere